<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>688</width>
    <height>622</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Simulator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Configuration</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QFrame" name="frame_2">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,1,0,0">
           <item>
            <widget class="QGroupBox" name="groupBox_6">
             <property name="title">
              <string>Load data</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_7">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_16">
                <item>
                 <widget class="QRadioButton" name="excel_radioButton">
                  <property name="text">
                   <string>Excel file</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QRadioButton" name="csv_radioButton">
                  <property name="text">
                   <string>Csv file</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_4">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QPushButton" name="load_pushButton">
                  <property name="text">
                   <string>Load file</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="title">
              <string>Data</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_18" stretch="0">
              <item>
               <widget class="QGroupBox" name="groupBox_3">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="title">
                 <string>Product selection</string>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_15">
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1">
                   <item>
                    <widget class="QLabel" name="label">
                     <property name="text">
                      <string>Betroffenes System</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QComboBox" name="systems_comboBox">
                     <property name="enabled">
                      <bool>true</bool>
                     </property>
                     <property name="editable">
                      <bool>false</bool>
                     </property>
                     <property name="currentText">
                      <string/>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="0,1">
                   <item>
                    <widget class="QLabel" name="label_2">
                     <property name="enabled">
                      <bool>true</bool>
                     </property>
                     <property name="text">
                      <string>Product (ZK2)</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QComboBox" name="product_comboBox">
                     <property name="enabled">
                      <bool>true</bool>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_2">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="title">
              <string>Models</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_17">
              <item>
               <widget class="QCheckBox" name="xgboost_checkBox">
                <property name="text">
                 <string>XGBoost </string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="catboost_checkBox">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>CatBoost</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_star_modeling">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="text">
                <string>Start modeling</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Result</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTabWidget" name="tabWidget_2">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab_3">
           <attribute name="title">
            <string>Plot</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_11" stretch="1,0">
            <item>
             <widget class="QWidget" name="widget" native="true"/>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_5">
              <property name="title">
               <string>Plots seting</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,0,1,0,1,0,1,0,1,0,0">
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_5">
                 <item>
                  <widget class="QLabel" name="label_3">
                   <property name="text">
                    <string>â„–</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_9">
                   <property name="text">
                    <string>1</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_10">
                   <property name="text">
                    <string>2</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <spacer name="horizontalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_6">
                 <item>
                  <widget class="QLabel" name="label_4">
                   <property name="text">
                    <string>New products</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QSpinBox" name="product_spinBox">
                   <property name="enabled">
                    <bool>true</bool>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>63</width>
                     <height>20</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QSpinBox" name="product_spinBox_2">
                   <property name="enabled">
                    <bool>true</bool>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>63</width>
                     <height>20</height>
                    </size>
                   </property>
                   <property name="value">
                    <number>1</number>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <spacer name="horizontalSpacer_3">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_8">
                 <item>
                  <widget class="QLabel" name="label_5">
                   <property name="text">
                    <string>New systems</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QSpinBox" name="systems_spinBox">
                   <property name="enabled">
                    <bool>true</bool>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>63</width>
                     <height>20</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QSpinBox" name="systems_spinBox_2">
                   <property name="enabled">
                    <bool>true</bool>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>63</width>
                     <height>20</height>
                    </size>
                   </property>
                   <property name="value">
                    <number>1</number>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <spacer name="horizontalSpacer_5">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_9">
                 <item>
                  <widget class="QLabel" name="label_6">
                   <property name="text">
                    <string>Number of active 
departments</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QSpinBox" name="department_spinBox">
                   <property name="enabled">
                    <bool>true</bool>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>63</width>
                     <height>20</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QSpinBox" name="department_spinBox_2">
                   <property name="enabled">
                    <bool>true</bool>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>63</width>
                     <height>20</height>
                    </size>
                   </property>
                   <property name="value">
                    <number>1</number>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <spacer name="horizontalSpacer_6">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_10">
                 <item>
                  <widget class="QLabel" name="label_7">
                   <property name="text">
                    <string>Dominant priority</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QSpinBox" name="priority_spinBox">
                   <property name="enabled">
                    <bool>true</bool>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>63</width>
                     <height>20</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QSpinBox" name="priority_spinBox_2">
                   <property name="enabled">
                    <bool>true</bool>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>63</width>
                     <height>20</height>
                    </size>
                   </property>
                   <property name="value">
                    <number>1</number>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <spacer name="horizontalSpacer_7">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QPushButton" name="pushButton_2">
                 <property name="text">
                  <string>Build</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_4">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <attribute name="title">
            <string>Statistics</string>
           </attribute>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>688</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
